LondonBritishLibrary/orient/BLorient713.xml

```diff
@@ -37,42 +37,61 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <msContents>
                         <summary/>
                         <msItem xml:id="ms_i1">
-                            <locus target="#4r">
+                            <title ref="LIT6172DossierGeorge"></title>
+                            
+                            <msItem xml:id="ms_i1.1">
+                            <locus from="4r">
                                 
                             </locus>
-                            <title type="complete" ref="LIT5464DiscourseofTheodotus">The discourse of Theodotus Ancyra</title>
+                                <title type="complete" ref="LIT1287Dersan">The discourse of Theodotus of Ankyra on the commemoration of St George</title>
                             <textLang mainLang="gez"/>
                         </msItem>
                         
                         <msItem xml:id="ms_i2">
-                            <locus target="#13r">
+                            <locus from="13r">
                                 
                             </locus>
-                            <title type="complete" ref="LIT1455GadlaG">The History and martyrdom of George</title>
+                            <title type="complete" ref="LIT1455GadlaG">The History and Martyrdom of George</title>
                             <textLang mainLang="gez"/>
+                            <colophon xml:id="coloph1" xml:lang="gez" corresp="LIT1455GadlaG#coloph1 BLorient712#coloph1">
+                                <locus target="#81r"/>
+                            </colophon>
+                            <colophon xml:id="coloph2" xml:lang="gez">
+                                <locus target="#81v"/>
+                                ወፍዳሜሁሰ፡ ኮነ፡፪ለወርኃ፡ ታስሪን፡ ቀደማይ፡ በሐሳቦሙ፡ ለዕብራውያን፡ ወበግዕዝ፡ አመ፡፱ዓውድ፡ ቀመር፡ ነ፩ጥንትዮን፡ ወበ፱ዓውደ፡ ዕለት። በ፱አበቅቴ፡ በካልአ፡ ዓመተ፡ መንግሥቱ፡ ለደዊት፡ ንጉሠ፡ አግዓዜ፡ ወልደ፡ ናዖድ፡ ንጉሠ፡ ድቱዓ፡ ሃይማኖት፡ በዕለተ፡ ሰኑይ፡ በ፲ሰዓት፡ ዘመዓልት፡ አመ፡ ከርመ፡ በምድረ፡ አዘዞ።
+                                <note>The copied colophon gives the precise date of translation, <date when="1510">2nd year of the reign of Dawit (that is, Lǝbna Dǝngǝl), who began his reign in August 1508</date> </note></colophon>
+                            <colophon xml:id="coloph3" xml:lang="gez">
+                                <locus target="#81v"/>
+                                <note>The note shows that the present copy was written for one <persName ref="PRS12580FessehaSeyon"/>, son of <persName ref="PRS12581Anorewos"/>, for which names have been substituted those of Walda ʾAragāwi and Walda ʾAragāwi.</note></colophon>
```

Comment:
What do you mean? There are two embedded colophons (work and translation, I will make them msItems) and then there is a note of the scribe, which is related to the MS, with names, which is the colophon, I think

Reply:
Ah, it seemed to me like the sentence belonged to the previous colophon. However, I would still be very cautious to specify that this note is a colophon. It might be a supplication or an ownership note ("shows that..." can mean a lot ).